client/asset/eth/eth.go

```diff
@@ -580,36 +579,11 @@ func createWallet(createWalletParams *asset.CreateWalletParams, skipConnect bool
 		}
 
 		if !skipConnect {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+			ctx, cancel := context.WithCancel(context.Background())
```

Comment:
What's the purpose of this cancelable context?

Reply:
Should I just use `context.Background()`?

Reply:
Using `context.Background()`